stop_line_list:
  -
    id : 1
    tl_id : 1
    pose :
      x : 1741.44
      y : 241.34
      z : -0.288
  -
    id : 2
    tl_id : 2
    pose :
      x : 1741.44
      y : 626.34
      z : -0.288
  -
    id : 3
    tl_id : 3
    pose :
      x : 1147.59
      y : 651.93
      z : -0.288
  -
    id : 4
    tl_id : 4
    pose :
      x : 1121.421
      y : 280.849
      z : -0.288